packages/cpp/src/browser/cpp-build-configurations.ts

```diff
@@ -54,8 +73,11 @@ export class CppBuildConfigurationManager {
     /** Emitter for when the active build configuration changes.  */
     protected readonly activeConfigChangeEmitter = new Emitter<CppBuildConfiguration | undefined>();
 
+    protected resource: Promise<Resource>;
```

Comment:
What is this for?  If this is really needed, it would require a better name and a comment.